src/Host.h

```diff
@@ -161,6 +161,7 @@ class Host  : public QObject
     bool               mAllowToSendCommand;
     bool               mAutoClearCommandLineAfterSend;
     bool               mBlockScriptCompile;
+    bool               mEchoLuaErrors; 
```

Comment:
There is no initialization value set in the `Host::Host(...)` constructor, whilst the line in the `(void) XMLimport::readHostPackage( Host * pT )` code will set it to `false` when it reads the profile in, I find that `bool` variables generally do not seem to assume a `false` value (watching the class members in the debugger in the Qt Creator IDE shows random NUMBER values for them) unless explicitly set that way in the constructor.

Avoid odd effects on profile loading with a line like:
`, mEchoLuaErrors( false )`
in the list under the line containing:
`Host::Host( int port, const QString& hostname, const QString& login, const QString& pass, int id )` in the `Host.cpp` file (not-obviously) it needs to be in the same order as the member variables are declared in the `Host.h` header file as the Clang analyser will otherwise moan about the out-of-order initialisation...  {if I'm preaching to someone who already knows this then you have my apologies but just in case you weren't then now you know!} :wink: